adhocteam/Head-Start-TTADP
# Add statistics widget

## Description of change

- Add statistics widget to the grantee record page */region/{regionId}/grantee/{granteeId}*

## How to test

- View the statistics widget on the grantee record page. Verify that the counts are correct and there are no pesky visual anomalies.

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-335


## Checklists

### Every PR


- [n/a] API Documentation updated
- [n/a] Boundary diagram updated
- [n/a] Logical Data Model updated
- [n/a] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions

### Production Deploy


### After merge/deploy
